File: Week6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break

ls = []
for i in range(height + 2):
    ls.append(" ")
for i in range(height):
    ls[height - 1 - i] = "#"
    ls.append("#")
    print("".join(ls))

File: Week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_slash(int height)
{
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - 1 - i; j++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}


int main(void)
{
    int height;
    do
    {
        height = get_int("please enter height\n");
    }
    while (height < 1 || height > 8);
    print_slash(height);
}